app/src/main/java/com/internship/move/feature/onboarding/OnboardingItem.kt

```diff
@@ -0,0 +1,12 @@
+package com.internship.move.feature.onboarding
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class OnboardingItem(
+    val imageURL: Int,
+    val titleText: String,
+    val bodyText: String,
+    val isSkipVisible: Boolean
```

Comment:
This field can have the default value `true`. This will simplify most of your constructor calls, since you'll only need to specify the value if it's `false`

Reply:
Also, personally I'd name it something like `isLastPage`, since you may want to use it for other things than just the skip button